core/src/main/java/tachyon/metrics/MetricsConfig.java

```diff
@@ -87,6 +87,11 @@ public Properties getInstanceProperties(String inst) {
    * Loads and parses the metrics configuration file.
    */
   private void initialize() {
+    // Set the default properties. The MetricsServlet is enabled and the path is /metrics/json
+    // by default.
+    mProperties.setProperty("*.sink.servlet.class", "tachyon.metrics.sink.MetricsServlet");
+    mProperties.setProperty("*.sink.servlet.path", "/metrics/json");
```

Comment:
`*.sink.servlet.class` means it is configured for both master and worker instances. Otherwise we can use `master.sink.servlet.class` and `worker.sink.servlet.class` to configure for master and worker respectively.